lesson28/src/js/register.js

```diff
@@ -98,3 +99,17 @@ const isRelatedTarget = (event,target) => {
     const related = event.relatedTarget ? event.relatedTarget.id : "unknown";
     return related === target;
 }
+
+const saveUserData = () => {
+    const userName = document.querySelector('[data-id="user_name"]');
+    const email = document.querySelector('[data-id="email"]');
+    const password = document.querySelector('[data-id="new-password"]');
+
+    const userData = {
+        name: userName.value,
+        email: email.value,
+        password : password.value,
+    }
+
+    window.localStorage.setItem("userData", JSON.stringify(userData));
+}
```

Comment:
Obtains the content entered on the member registration page and stores it in an object. 
The object is converted to a JSON string and stored in LocalStrage.